swift_browser_ui/api.py

```diff
@@ -15,46 +15,39 @@
 from .settings import setd
 
 
-async def get_os_user(
-        request: aiohttp.web.Request
-) -> aiohttp.web.Response:
+async def get_os_user(request: aiohttp.web.Request) -> aiohttp.web.Response:
     """Fetch the session owning OS user."""
     session = api_check(request)
-    request.app['Log'].info(
-        'API call for username from {0}, sess: {1} :: {2}'.format(
+    request.app["Log"].info(
+        "API call for username from {0}, sess: {1} :: {2}".format(
```

Comment:
f strings are also an option, considering that in some places we use f string and in some `.format` we need to pick one. I think f strings provider a cleaner syntax but also ok with `.format`. The same in other places.

Reply:
moving over to f-strings with next push to this branch – unifying string style is a good thing